Fri, 26 Jan 2024 15:02:19-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Fri, 26 Jan 2024 15:02:41-RVPlayView-table_operation.py-[line:75]-WARNING-[日志信息]: Capturing images takes time: 1.7095553874969482
Fri, 26 Jan 2024 15:10:01-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Fri, 26 Jan 2024 15:10:29-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Fri, 26 Jan 2024 15:10:48-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Fri, 26 Jan 2024 15:11:49-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Fri, 26 Jan 2024 15:12:59-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Fri, 26 Jan 2024 15:13:16-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Fri, 26 Jan 2024 15:13:34-RVPlayView-table_operation.py-[line:75]-WARNING-[日志信息]: Capturing images takes time: 1.5950343608856201
Thu, 01 Feb 2024 12:48:32-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Thu, 01 Feb 2024 12:48:43-RVPlayView-table_operation.py-[line:75]-WARNING-[日志信息]: Capturing images takes time: 3.03882098197937
Thu, 01 Feb 2024 12:48:58-RVPlayView-table_operation.py-[line:75]-WARNING-[日志信息]: Capturing images takes time: 0.5394716262817383
Thu, 01 Feb 2024 12:49:58-RVPlayView-table_operation.py-[line:75]-WARNING-[日志信息]: Capturing images takes time: 0.8210005760192871
Thu, 01 Feb 2024 13:05:42-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Thu, 01 Feb 2024 13:06:04-RVPlayView-table_operation.py-[line:75]-WARNING-[日志信息]: Capturing images takes time: 0.0
Fri, 02 Feb 2024 17:09:17-RVPlayView-table_operation.py-[line:75]-WARNING-[日志信息]: Capturing images takes time: 2.546043634414673
Fri, 02 Feb 2024 17:33:59-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Fri, 02 Feb 2024 17:34:20-RVPlayView-table_operation.py-[line:75]-WARNING-[日志信息]: Capturing images takes time: 2.1701130867004395
Fri, 02 Feb 2024 17:34:53-RVPlayView-table_operation.py-[line:98]-WARNING-[日志信息]: Obtaining all video frames takes time: 16.983773469924927
Fri, 02 Feb 2024 17:35:46-RVPlayView-table_operation.py-[line:75]-WARNING-[日志信息]: Capturing images takes time: 2.6153881549835205
Fri, 02 Feb 2024 17:40:31-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Fri, 02 Feb 2024 17:42:08-RVPlayView-table_operation.py-[line:75]-WARNING-[日志信息]: Capturing images takes time: 89.46081805229187
Fri, 02 Feb 2024 17:47:57-RVPlayView-table_operation.py-[line:98]-WARNING-[日志信息]: Obtaining all video frames takes time: 2.869431257247925
Fri, 02 Feb 2024 17:49:03-RVPlayView-table_operation.py-[line:98]-WARNING-[日志信息]: Obtaining all video frames takes time: 1.893146276473999
Fri, 02 Feb 2024 17:50:46-RVPlayView-table_operation.py-[line:98]-WARNING-[日志信息]: Obtaining all video frames takes time: 4.2218337059021
Sun, 04 Feb 2024 19:18:43-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Sun, 04 Feb 2024 19:18:59-RVPlayView-table_operation.py-[line:79]-WARNING-[日志信息]: Capturing images takes time: 7.520931005477905
Sun, 04 Feb 2024 19:22:58-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Sun, 04 Feb 2024 19:23:17-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 10.98446273803711
Sun, 04 Feb 2024 19:23:53-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Sun, 04 Feb 2024 19:24:06-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 2.525569200515747
Sun, 04 Feb 2024 19:24:30-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Sun, 04 Feb 2024 19:24:41-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 2.5605695247650146
Sun, 04 Feb 2024 19:31:45-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Sun, 04 Feb 2024 19:31:55-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 2.5957906246185303
Sun, 04 Feb 2024 19:33:22-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Sun, 04 Feb 2024 19:35:17-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Sun, 04 Feb 2024 19:35:50-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 24.739198684692383
Sun, 04 Feb 2024 19:36:32-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Sun, 04 Feb 2024 19:36:43-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 3.0596859455108643
Sun, 04 Feb 2024 19:37:24-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.03350567817687988
Sun, 04 Feb 2024 19:37:45-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 1.3530373573303223
Sun, 04 Feb 2024 19:37:59-RVPlayView-table_operation.py-[line:106]-WARNING-[日志信息]: Obtaining all video frames takes time: 6.667625665664673
Sun, 04 Feb 2024 19:37:59-RVPlayView-table_operation.py-[line:106]-WARNING-[日志信息]: Obtaining all video frames takes time: 11.089205980300903
Sun, 04 Feb 2024 19:38:03-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.40732431411743164
Sun, 04 Feb 2024 19:38:10-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.017464637756347656
Sun, 04 Feb 2024 19:38:16-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.008976459503173828
Sun, 04 Feb 2024 19:38:35-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.05396842956542969
Sun, 04 Feb 2024 19:38:42-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.002992391586303711
Tue, 27 Feb 2024 15:03:28-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 15:03:50-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.013733386993408203
Tue, 27 Feb 2024 15:04:10-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 4.945782661437988
Tue, 27 Feb 2024 15:07:48-RVPlayView-table_operation.py-[line:106]-WARNING-[日志信息]: Obtaining all video frames takes time: 9.824122667312622
Tue, 27 Feb 2024 15:08:08-RVPlayView-table_operation.py-[line:106]-WARNING-[日志信息]: Obtaining all video frames takes time: 1.095721960067749
Tue, 27 Feb 2024 15:30:32-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 15:30:49-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 15:31:03-RVPlayView-RVToolMain.py-[line:31]-ERROR-[日志信息]: 程序BUG日志
Traceback (most recent call last):
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 646, in create_table
    self.add_table(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 639, in add_table
    self.stacked_layout.addWidget(TableItem(session, db_entity))
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 580, in __init__
    self.setup(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 593, in setup
    self.table = RVTable(session, db_entity, self.status_bar)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 246, in __init__
    self.setModel(RVTableModel())
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 77, in headerData
    return self.__stage_list[section]
IndexError: list index out of range
Tue, 27 Feb 2024 15:31:45-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 15:32:04-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 2.012946605682373
Tue, 27 Feb 2024 15:54:23-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 15:54:37-RVPlayView-RVToolMain.py-[line:31]-ERROR-[日志信息]: 程序BUG日志
Traceback (most recent call last):
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 651, in create_table
    self.add_table(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 644, in add_table
    self.stacked_layout.addWidget(TableItem(session, db_entity))
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 585, in __init__
    self.setup(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 598, in setup
    self.table = RVTable(session, db_entity, self.status_bar)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 251, in __init__
    self.setModel(RVTableModel())
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 84, in headerData
    return self.__source_data[section][0].get_shot_entity(is_package=True, repeat=True)
IndexError: list index out of range
Tue, 27 Feb 2024 15:56:02-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 15:56:09-RVPlayView-RVToolMain.py-[line:31]-ERROR-[日志信息]: 程序BUG日志
Traceback (most recent call last):
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 652, in create_table
    self.add_table(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 645, in add_table
    self.stacked_layout.addWidget(TableItem(session, db_entity))
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 586, in __init__
    self.setup(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 599, in setup
    self.table = RVTable(session, db_entity, self.status_bar)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 252, in __init__
    self.setModel(RVTableModel())
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 85, in headerData
    return self.__source_data[section][0].get_shot_entity(is_package=True, repeat=True)
IndexError: list index out of range
Tue, 27 Feb 2024 15:56:52-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 15:57:02-RVPlayView-RVToolMain.py-[line:31]-ERROR-[日志信息]: 程序BUG日志
Traceback (most recent call last):
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 652, in create_table
    self.add_table(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 645, in add_table
    self.stacked_layout.addWidget(TableItem(session, db_entity))
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 586, in __init__
    self.setup(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 599, in setup
    self.table = RVTable(session, db_entity, self.status_bar)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 252, in __init__
    self.setModel(RVTableModel())
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 85, in headerData
    return self.__source_data[section][0].get_shot_entity(is_package=True, repeat=True)
IndexError: list index out of range
Tue, 27 Feb 2024 15:57:48-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 15:57:55-RVPlayView-RVToolMain.py-[line:31]-ERROR-[日志信息]: 程序BUG日志
Traceback (most recent call last):
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 653, in create_table
    self.add_table(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 646, in add_table
    self.stacked_layout.addWidget(TableItem(session, db_entity))
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 587, in __init__
    self.setup(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 600, in setup
    self.table = RVTable(session, db_entity, self.status_bar)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 252, in __init__
    self.setModel(RVTableModel())
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 85, in headerData
    return self.__source_data[section][0].get_shot_entity(is_package=True, repeat=True)
IndexError: list index out of range
Tue, 27 Feb 2024 15:58:23-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 15:58:30-RVPlayView-RVToolMain.py-[line:31]-ERROR-[日志信息]: 程序BUG日志
Traceback (most recent call last):
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 654, in create_table
    self.add_table(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 647, in add_table
    self.stacked_layout.addWidget(TableItem(session, db_entity))
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 588, in __init__
    self.setup(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 601, in setup
    self.table = RVTable(session, db_entity, self.status_bar)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 253, in __init__
    self.setModel(RVTableModel())
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 86, in headerData
    return self.__source_data[section][0].get_shot_entity(is_package=True, repeat=True)
IndexError: list index out of range
Tue, 27 Feb 2024 16:00:02-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 16:00:08-RVPlayView-RVToolMain.py-[line:31]-ERROR-[日志信息]: 程序BUG日志
Traceback (most recent call last):
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 653, in create_table
    self.add_table(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 646, in add_table
    self.stacked_layout.addWidget(TableItem(session, db_entity))
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 587, in __init__
    self.setup(session, db_entity)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 600, in setup
    self.table = RVTable(session, db_entity, self.status_bar)
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 252, in __init__
    self.setModel(RVTableModel())
  File "D:\TA_CGTools\CGTeamWork\Plugins\PlayView\rv_widgets\rv_table.py", line 85, in headerData
    return self.__source_data[section][0].get_shot_entity(is_package=True, repeat=True)
IndexError: list index out of range
Tue, 27 Feb 2024 16:01:40-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 16:01:47-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.024882078170776367
Tue, 27 Feb 2024 16:02:08-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 16:02:16-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.036673545837402344
Tue, 27 Feb 2024 16:02:46-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 16:02:56-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.033908843994140625
Tue, 27 Feb 2024 16:03:21-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 16:05:55-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.024624109268188477
Tue, 27 Feb 2024 16:07:28-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 6.805253744125366
Tue, 27 Feb 2024 16:08:54-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 16:11:27-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.7532444000244141
Tue, 27 Feb 2024 16:11:36-RVPlayView-table_operation.py-[line:106]-WARNING-[日志信息]: Obtaining all video frames takes time: 2.748603582382202
Tue, 27 Feb 2024 16:11:47-RVPlayView-table_operation.py-[line:106]-WARNING-[日志信息]: Obtaining all video frames takes time: 1.1784067153930664
Tue, 27 Feb 2024 16:16:00-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.008975744247436523
Tue, 27 Feb 2024 16:21:00-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 16:22:29-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.02194046974182129
Tue, 27 Feb 2024 16:26:24-RVPlayView-RVToolMain.py-[line:27]-WARNING-[日志信息]: ['D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\python39.zip', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\DLLs', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39', 'C:\\Users\\Administrator\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\pyside6-6.6.0-py3.9-win-amd64.egg', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\win32\\lib', 'C:\\ProgramData\\Anaconda3\\envs\\Py39\\lib\\site-packages\\Pythonwin', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\rvcore/network', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView', 'C:/CgTeamWork_v7/bin/base', 'D:\\TA_CGTools\\CGTeamWork\\Plugins\\PlayView\\Lib/site-packages']
Tue, 27 Feb 2024 16:27:07-RVPlayView-table_operation.py-[line:83]-WARNING-[日志信息]: Capturing images takes time: 0.6876523494720459
